examples/README.md

```diff
@@ -1,18 +1,18 @@
 # Determined Examples
 
+Please find a large collection of community-sourced examples in
+[determined-ai/determined-examples](https://github.com/determined-ai/determined-examples/) repo.
+
+This directory only contains the examples which are routinely tested on CI.
```

Comment:
I'd swap the order here.

"This directory only contains examples that are routinely tested on our CI platform. Further examples are located in our community-sourced [determined-ai/determined-examples](https://github.com/determined-ai/determined-examples/) repo."